src/screens/LoginScreen/components/LoginForm/components/FormField/styles.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import { themeGet } from "@styled-system/theme-get";
+
+export const ErrorMessageText = styled.small`
+  font-size: 11px;
+  color: ${themeGet("colors.error")};
+  display: block;
+`;
+
+export const InputClass = {
```

Comment:
This is a cool solution, but you can also use `styled` on non-builtin components. You can import the `Field` component in the `styles.js` and then you can do `export const InputClass = styled(Field)\`bla bla bla\``, what this does under the hood is the exact same thing you did. It convers the CSS code into an object and passes the object as a `styles` prop to the component, so you don't have to write the object yourself you can just basically write CSS code :)